app/views/static/buttons/generic_button_component.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
I think a separate PR is a perfect idea ;). The component should be able to work in *any* form, that's the point of passing everything in explicitly... So if the component can't work with non-converted form, it's not done right ;).